tracpro/baseline/views.py

```diff
@@ -77,7 +77,14 @@ def get_context_data(self, **kwargs):
             context = super(BaselineTermCRUDL.Read, self).get_context_data(**kwargs)
 
             # Get the region from the region filter drop-down, if it was selected
-            region = int(self.request.GET.get('region', 0))
+            try:
+                region = int(self.request.GET.get('region', 0))
+            except:
```

Comment:
Let's narrow this to `ValueError`.

Casting as an `int` can also lead to a `TypeError` if `None` is passed - but that shouldn't ever come into play here.